clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -157,8 +157,10 @@ public class AsyncKafkaConsumerTest {
     @AfterEach
     public void resetAll() {
         backgroundEventQueue.clear();
-        if (consumer != null) {
+        try {
             consumer.close(Duration.ZERO);
+        } catch (Exception e) {
+            // ignore
```

Comment:
I'm a little leery about swallowing the exception here. Can we validate the exception type is something we expect? e.g.:

```suggestion
        } catch (Exception e) {
            assertInstanceOf(KafkaException.class, e);
```

Reply:
Done